src/game.py

```diff
@@ -0,0 +1,41 @@
+from src.cli import Cli
+from src.board import Board
+from src.player import Player
+
+class Game():
+
+    def __init__(self, cli=Cli(), board=Board(), player_1=Player(token="X"),
+                 player_2=Player(token="O")):
+        self.cli = cli
+        self.board = board
+        self.player_1 = player_1
+        self.player_2 = player_2
+
+    def start(self):
+        self.cli.welcome()
+        self.play()
+
+    def play(self):
+        self.cli.log(self.board.stringify_spaces())
+        while not self.board.full():
+            self.turn()
+
+        self.cli.goodbye()
+    
+    def turn(self):
+        player_choice = self.cli.prompt_user()
+        move = self.input_to_index(player_choice)
+
+        self.board.update(move, self.current_player().token)
+        self.cli.log(self.board.stringify_spaces())
+       
+    def input_to_index(self, user_input):
+        input_int = int(user_input)
+        return input_int - 1 
+
+    def current_player(self):
+        if self.board.turn_count() % 2 == 0:
```

Comment:
What if the number of players changes in the future?

Reply:
I'm not sure... I suppose I should look for a better way to keep track of the current player if that's something that could possibly change in the future. 

Candidly, I never really considered it

Reply:
Typically if a feature isn't "future-proofed" it just means it isn't encapsulated well. I think the concept of a player count isn't encapsulated well at present.

Reply:
So should I keep track of the playercount as a constant like we did for the board's size? My concern with that is that this current_player would just break if that constant was changed. Do I just need to completely re-address how I'm determining the current_player then so that it would work better if we chose to change the player_count at some point in the future?

Reply:
The % aspect of it would be good, but yeah, the way you have the if statements needs to change. Try to figure out a way that doesn't require additional if statements each time you add a player.

Reply:
Alright I'll chew on that for a while and try and work out a working solution. Thanks

Reply:
I don't really see how % helps if there is more than two players, the reason it works with two players right now is because the result is either even or odd and depending on that result I can tell whose turn is next. Do you have any blog posts or anything that you can point me to that talks about how to approach this?

Alternatively I was thinking maybe we can store the players into a list on the game and use the turn count to operate on the list to determine whose turn it is. Should I scrap how I'm determining player_count completely and go with a strategy more like this?

Reply:
Array of players + % == victory

Reply:
Great, ill work on it